Kube API Test dependency not resolvable

### Describe the bug

Hello, I'm trying to use the Kube API Test artifact in a project, but Maven is unable to resolve the dependency. 

I see that in the New Features section of the [release notes of **6.11.0** ](https://github.com/fabric8io/kubernetes-client/releases/tag/v6.11.0) the Kube API Test is mentioned, but nonetheless the artifact isn't available in the [Maven central repositoty](https://repo.maven.apache.org/maven2/). I've found [this issue](https://github.com/fabric8io/kubernetes-client/issues/5819) in which it was also mentioned the the feature should come in **6.11.0**.

I also tried to use the **6.12.0** version, but the same problematic behavior is observed. 

### Fabric8 Kubernetes Client version

6.12.0

### Steps to reproduce

1. Add the following dependency to a pom.xml file:
```
        <dependency>
            <groupId>io.fabric8</groupId>
            <artifactId>kube-api-test</artifactId>
            <version>6.12.0</version>
            <scope>test</scope>
        </dependency>
```
2. Run `mvn clean install`
3. The following error is observed from the command output:
```
[ERROR] Failed to execute goal on project kube-api-server-test: Could not resolve dependencies for project org.example:kube-api-server-test:jar:1.0-SNAPSHOT: Could not find artifact io.fabric8:kube-api-test:jar:
6.12.0 in central (https://repo.maven.apache.org/maven2) -> [Help 1]
```

### Expected behavior

Dependency is downloaded and project build is successful.

### Runtime

Kubernetes (vanilla)

### Kubernetes API Server version

other (please specify in additional context)

### Environment

Windows, other (please specify in additional context)

### Fabric8 Kubernetes Client Logs

_No response_

### Additional context

K8s version: irrelevant

--- a/.github/workflows/release-snapshots.yaml
+++ b/.github/workflows/release-snapshots.yaml
@@ -89,7 +89,7 @@ jobs:
           gpg-private-key: ${{ secrets.SIGNINGKEY }}
           gpg-passphrase: SIGNINGPASSWORD
       - name: Build and release Java 11 modules
-        run: ./mvnw ${MAVEN_ARGS} ${RELEASE_MAVEN_ARGS} -pl "httpclient-jdk" -pl "httpclient-jetty" clean deploy
+        run: ./mvnw ${MAVEN_ARGS} ${RELEASE_MAVEN_ARGS} -pl "httpclient-jdk" -pl "httpclient-jetty" -pl "junit/kube-api-test" clean deploy
       - name: Build and release BOM
         run: |
           ./mvnw ${MAVEN_ARGS} ${RELEASE_MAVEN_ARGS} clean sundr:generate-bom                                                                                                                                                                                                                                                                         &&

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -98,7 +98,7 @@ jobs:
           gpg-private-key: ${{ secrets.SIGNINGKEY }}
           gpg-passphrase: SIGNINGPASSWORD
       - name: Build and release Java 11 modules
-        run: ./mvnw ${MAVEN_ARGS} ${RELEASE_MAVEN_ARGS} -pl "httpclient-jdk" -pl "httpclient-jetty" clean deploy
+        run: ./mvnw ${MAVEN_ARGS} ${RELEASE_MAVEN_ARGS} -pl "httpclient-jdk" -pl "httpclient-jetty" -pl "junit/kube-api-test"  clean deploy
       - name: Build and release BOM
         run: |
           ./mvnw ${MAVEN_ARGS} ${RELEASE_MAVEN_ARGS} clean sundr:generate-bom                                                                                                                                                                                                                                                                              &&

